subprojects/docs/src/snippets/testing/test-suite-multi-configure-each-matching/kotlin/build.gradle.kts

```
@@ -0,0 +1,62 @@
+import org.gradle.api.internal.tasks.testing.junit.JUnitTestFramework
+
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    `java-library`
+}
+
+version = "1.0.0"
+group = "org.gradle.sample"
+
+repositories {
+    mavenCentral()
+}
+
+// tag::multi-configure[]
+testing {
+    suites.withType(JvmTestSuite::class).matching { listOf("test", "integrationTest").contains(it.name) }.configureEach { // <1>
+        useJUnitJupiter()
+        dependencies {
+            implementation("org.mockito:mockito-junit-jupiter:4.6.1")
+        }
+    }
+
+    suites {
```

To make this more like the Groovy version, I would move this into the suites {}
```suggestion
    suites {
        withType(JvmTestSuite::class).matching { it.name in listOf("test", "integrationTest") }.configureEach { // <1>
            useJUnitJupiter()
            dependencies {
                implementation("org.mockito:mockito-junit-jupiter:4.6.1")
            }
        }
```